SWLOR.Game.Server/Service/Communication.cs

```diff
@@ -461,6 +501,10 @@ public static void ProcessChatMessage()
                 sender = originalSender;
 
                 ChatPlugin.SendMessage(finalChannel, finalMessageColoured, sender, obj);
+                var msgFCL = langMessageForCombatLog.ToString();
```

Comment:
Try to avoid acronyms - use something like messageForCombatLog or similar. It's more typing but it helps with readability.